README.md

```diff
@@ -184,9 +184,9 @@ HTTP Basic authentication (or "Basic Auth") has been around for quite some time.
 **Some important notes on HTTP Basic Authentication:**
 
 - The example authorization header above does not look like a username and password, but that is because it is base64 encoded. IT IS NOT ENCRYPTED.
-- USE HTTPS if you are using Basic Auth. If you just use HTTP, authentication credentials are sent to a server as PLAIN TEXT. THIS IS BAD. A user's username and password are being sent over the wire merely as base64 encoded text — which is trivial to decode. By using HTTPS / TLS you are ensuring that data sent from the client to the server is encrypted.
+- USE HTTPS if you are using Basic Auth. If you just use HTTP, authentication credentials are sent to a server as PLAIN TEXT. THIS IS BAD. A user's username and password are being sent over the wire merely as base64 encoded text — which is trivial to decode. By using HTTPS / TLS, you are ensuring that data sent from the client to the server is encrypted.
 - Basic Auth, _as implemented by the browser_, is not exactly aesthetically pleasing and is rarely used today.
-- Basic Auth with APIs, when combined with tokens, (discussed later) is just an `Authorization` header and is perfectly reasonable. It has the added benefit of not requiring the API client to maintain an extra session cookie and, since most systems log query params but not headers, will not be logged by default.
+- Basic Auth with APIs, when combined with tokens, (discussed later) is just an `Authorization` header and is perfectly reasonable. It has the added benefit of not requiring the API client to maintain an extra session cookie and, since most systems, log query params but not headers, will not be logged by default.
```

Comment:
I think the previous punctuation was correct. The parenthetical phrase here is `since most systems log query params but not headers`.